cip-0008.md

```diff
@@ -0,0 +1,33 @@
+
+Before a CIP custodian may provision a donation address for a specific CIP, that CIP must be written to include a section entitled "Milestones", which shall include the following:
+
+* The fundraising goal, being a total amount (expressed in BTC, XCP or USD) that must be raised in order to consider fundraising for the CIP complete. (Contributions over and above this amount will be refunded to their senders by the CIP custodian.)  
+* Two or more discrete milestones, and for each list specific criteria which must be met in order to consider the milestone complete.
+* Each milestone must also list a certain percentage figure, which will be the percentage of the total funds raised that will be disbursed by the CIP custodian upon the listed criteria being satisfied. (Any "kickoff" milestone present may not exceed 10% payment.)
+
+Once produced, the donation address will be provided back to the CIP author over a pubic, auditable medium such as the Counterparty forums. The CIP author shall then include it in the "Milestones" section of the CIP draft, and may then commence the fundraising necessary to attract development talent to implement the CIP for an amount up to the listed fundraising goal. Once an interested developer is found, that developer's name should also be listed in the CIP as the individual (or individuals) implementing it. The CIP custodian shall not serve as a developer for any CIP for which he or she has control (or partial control) of the funds.
+
+The CIP custodian is only obligated to disburse funds once the CIP is approved *and* the appropriate milestones are completed. If either is not the case, no funds will be disbursed. If the CIP is rejected or deferred, the CIP author may choose to ask the CIP custodian to refund the raised funds to the community. In this event, the CIP custodian will send the funds back to each address that sent them, minus any applicable transaction fees. (For this reason, individuals should never donate from an exchange address, and should only donate from an address they control.)
+
+The decision as to whether a milestone is reached shall be the sole decision of a project maintainer (i.e. an individual that has commit access to the [counterparty-lib]( https://github.com/CounterpartyXCP/counterparty-lib) repository). In the event of fraud or sufficient controvercy surrounding a CIP or CIP bounty, a project maintainer or the Counterparty Foundation board (via a majority vote) may compel the CIP custodian to refund undisbursed funds.
```

Comment:
Which project maintainer?  What if Robby says yes and Ruben says no?

Should the project maintainer who approves a milestone be listed in the CIP?  Should it require a majority vote (which would be both right now)?